.ci/setup.sh

```diff
@@ -22,7 +22,10 @@
 set -e
 
 test_repo="github.com/clearcontainers/tests"
+packaging_repo="github.com/clearcontainers/packaging"
 
+# Clone packaging repository.
+go get "$test_packaging"
```

Comment:
should we do this on the `tests` repo? If we need it here, we might need it also in the shim, proxy and other repos...

Reply:
@jcvenegas - the description suggests this commit might need to be deleted?